test/v1/connectors/RetrieveEmploymentsExpensesConnectorSpec.scala

```diff
@@ -17,71 +17,53 @@
 package v1.connectors
 
 import mocks.MockAppConfig
+import v1.fixtures.RetrieveEmploymentsExpensesFixtures._
 import v1.mocks.MockHttpClient
 import v1.models.domain.{MtdSource, Nino}
-import v1.models.downstream.DownstreamSource
 import v1.models.outcomes.ResponseWrapper
+import v1.models.request.TaxYear
 import v1.models.request.retrieveEmploymentExpenses.RetrieveEmploymentsExpensesRequest
-import v1.models.response.retrieveEmploymentExpenses.{Expenses, RetrieveEmploymentsExpensesResponse}
 
 import scala.concurrent.Future
 
 class RetrieveEmploymentsExpensesConnectorSpec extends ConnectorSpec {
 
-  val nino: String                  = "AA123456A"
-  private val taxYear               = "2019-20"
-  val source: MtdSource.`user`.type = MtdSource.`user`
+  val nino: String      = "AA123456A"
+  val source: MtdSource = MtdSource.`user`
 
-  class Test extends MockHttpClient with MockAppConfig {
+  def makeRequest(taxYear: String): RetrieveEmploymentsExpensesRequest =
+    RetrieveEmploymentsExpensesRequest(Nino(nino), TaxYear.fromMtd(taxYear), source)
+
+  val nonTysRequest = makeRequest("2019-20")
+  val tysRequest    = makeRequest("2023-24")
+
+  trait Test extends MockHttpClient with MockAppConfig {
```

Comment:
Should we make this `trait Test { _: ConnectorTest =>` just to be consistent with other api endpoints?